### PR TITLE
build(client): Tag asserts

### PR DESCRIPTION
**Pull request by tylerbutler**
Tagging asserts in preparation for a release.

Commands used:

```shell
npm run policy-check:asserts
```

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -6640,6 +6640,12 @@
               }
             }
           }
+        },
+        "tslib": {
+          "version": "2.4.1",
```
Comment (CraigMacomber) [inline package-lock.json:6645]: bunch of unrelated changes in here.

Comment (tylerbutler) [reply package-lock.json:6645]: whoops... Will revert.

**Review by CraigMacomber — APPROVED** [review]
